libevmasm/Assembly.cpp

```diff
@@ -74,6 +75,123 @@ unsigned Assembly::codeSize(unsigned subTagSize) const
 	}
 }
 
+void Assembly::addAssemblyItemsFromJSON(Json::Value const& _code)
+{
+	solAssert(m_items.empty());
+	solAssert(_code.isArray());
+	for (auto const& jsonItem: _code)
+		m_items.emplace_back(createAssemblyItemFromJSON(jsonItem));
+
+	for (auto current = m_items.begin(); current != m_items.end(); ++current)
+	{
+		// During the assembly json export a `JUMPDEST` is always generated after a `tag`.
+		// So we just ignore exactly these `JUMPDEST`'s.
+		auto const next = std::next(current);
```

Comment:
```suggestion
		auto const next = next(current);
```
I so wish we used `clang-format` and didn't have to do this :D

Reply:
True, the problem here was just that the variable `next` shadowed `next` (the iterator function). Now I just changed the variable name.

However, is there a special configuration in `clang-format` for doing this automatically? We have in our repository root a `.clang-format` file that can be used by some IDEs. Would it make sense to add such config there (another PR of course)?

Reply:
Don't get distracted by clang-format :-). We'll discuss this again eventually and probably rather sooner than later, but we shouldn't be jumping at it beforehand :-).

Reply:
Yeah we do need to discuss it, but short answer to @aarlt's question is no - unless we conform to a certain base style that clang-format insists on (e.g. LLVM, Google, Microsoft, etc.), and then configure desired overrides, we're not gonna get what we want. And what we want is not possible as none of the base style support it; so ultimately, we'll have to decide whether to simply not use clang-format ever, or to adapt.